.rubocop.yml

```diff
@@ -10,7 +10,7 @@ AllCops:
 Layout/LineLength:
   Max: 120
 Metrics/MethodLength:
-  Max: 20
+  Max: 40
```

Comment:
Please revert this change since we have been working with these settings since the start of the project.